.travis.yml

```diff
@@ -11,16 +12,24 @@ services:
   - postgresql
 before_install:
   - rvm install ruby-1.9.3-p551
+  # Repo for Yarn
+  - sudo apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg
+  - echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+  - sudo apt-get update -qq
+  - sudo apt-get install -y -qq yarn=$YARN_VERSION
```

Comment:
Travis does have yarn pre-installed, but it is an old version and I don't think there is a way to specify a newer one. I got these install script lines from https://yarnpkg.com/en/docs/install-ci#travis-tab

Unfortunately, that means `yarn` is installed twice with every Travis run. But it doesn't add that much to the build time, and we'll be switching to Circle soon enough so I think we shouldn't worry too much about it.